markdown/2017-10-02-welcome.md

```diff
@@ -0,0 +1,380 @@
+on the event, just ask a committee member to give you a few more tokens. We
+don't bite!
+
+### Board Game Nights
+
+Whether you're a *Monopoly* person or a *Settlers of Catan* person, board game
+nights are the perfect opportunity to get your nerd on and fight for victory
+with free snacks and drinks all evening.
+
+### Talks
+Our talks usually happen in the evenings, just after lectures. If you attended a
+talk around lunch time and didn't like it, you're most likely confusing the
+department's ACI talks with our talks. At the end of a talk we usually hand out
+pizza for everyone! Veggie options are of course available.
+
+### IC Hack
+We run a hackathon every year called IC Hack.
+
+Check out the epicness of IC Hack 2017 on
+[YouTube](https://www.youtube.com/watch?v=VQl8tLcxptY)!
+
+Want to get participate? Want to help run IC Hack? Details will follow later this
+term!
+We encourage all freshers to participate in their first year. Hackathons are a
+blast to attend and an amazing experience regardless of whether you've never
+been to a hackathon before or attend a different one very week.
+
+---
+
+Now for all the juicy details on our events this week(ish):
+
+# Freshers' Fair
+
+- **When:** <strong><font color="#ff0000">TOMORROW!</font></strong> 11.00-16.00, 3rd October 2017
+- **Where:** ICME10 (Main Entrance)
+
+Come and visit us at our stall at the Freshers' Fair to learn more about us,
+chat with the committee, eat lots of sweets, and win some great prizes in our
+speed typing competition! We'll be there all day so make sure to check out all
+of the other fantastic opportunities as well! You can find us by consulting the
+official map.
+
+The speed typing prizes are (in no particular order):
+- GoPro HERO5 Session
+- 25" 21:9 LG UltraWide Monitor
+- Amazon Echo
+
+# Buddies Lunch
+
+- **When:** 12.30-14.00, 9th October 2017
+- **Where:** Huxley 341/342
+
+As a fresher, you will have already heard of our Mums and Dads (MaD) programme
+from your department representatives. If you don't have any parents assigned yet
+or have never heard of this programme, please reach out to your department
+representatives (above).
+
+The buddies lunch will be an opportunity for you to meet your MaD parents and
+have lunch with them! We will be offering free food and drink and issuing a
+number of challenges. You might even win a prize! And if you're a parent, you'll
+get to meet your children!
+
+# Sponsors' Exhibition
+
+- **When:** 14.00-17.30, 12th October 2017
+- **Where:** The Great Hall
+- **Link:** [Facebook](https://www.facebook.com/events/1852626871714356/)
+
+Looking for an internship? Or want to talk directly to industry and our fabulous
+sponsors? Come join us in the Great Hall to have a chat with companies ranging
+from Facebook to Palantir and Netcraft to Microsoft in our premier autumn event.
+Bring lots of questions and be prepared to go home with a lot of swag!
+
+# Bloomberg CodeCon
+
+- **When:** 18.00-20.00, 13th October 2017
```

Comment:
Shift this by half an hour. 18:30-20:30

Reply:
I was using what was on the FB event which btw needs to be published once the digest is released https://www.facebook.com/events/1816220712002800/